packages/shared-business/src/utils/iban.ts

```diff
@@ -0,0 +1,9 @@
+import { isValid as isValidIban } from "iban";
+import { t } from "./i18n";
+export { printFormat as printIbanFormat } from "iban";
+
+export const validateIban = (iban: string) => {
+  if (!isValidIban(iban)) {
+    return t("error.iban.invalid");
+  }
+};
```

Comment:
Maybe put this in https://github.com/swan-io/lake/blob/main/packages/shared-business/src/utils/validation.ts ?